### ISSUE TITLE
Running bidscoiner.py without bidsmapper gives puzzling error

### ISSUE THREAD

**Issue by TimVanMourik** [Donders-Institute/bidscoin#48]
Plainly running:
```
bidscoiner.py [sourcefolder] [bidsfolder]
```
without first running the bidsmapper gives an error message:
```
No bidsmap file found: /opt/bidscoin/dev/heuristics/bidsmap.yaml
```
Which indeed there isn't:
```
$ ls -l /opt/bidscoin/2.3/heuristics/
total 48
-rw-r--r-- 1 marzwi mrphys 21065 Sep 19 18:53 bidsmap_dccn.yaml
-rw-r--r-- 1 marzwi mrphys 17141 Sep 19 18:53 bidsmap_template.yaml
-rw-r--r-- 1 marzwi mrphys    19 Aug 25 12:12 __init__.py
```

It would be nice to get an error that suggests to first run the bidsmapper. This error lead me to think that there might be a file internal to the bidsmapper that was missing.

**Comment by marcelzwiers** [comment]
Solved with this commit: 05c95e67fd2e65a58237d29675358c2d3638a530

**Comment by TimVanMourik** [comment]
Nice, thanks!
